src/Core/Commander/Providers/TileProvider.js

```diff
@@ -44,6 +47,7 @@ TileProvider.prototype.executeCommand = function executeCommand(command) {
         bbox,
         level: (command.level === undefined) ? (parent.level + 1) : command.level,
         segment: 16,
+        light: command.layer.light,
```

Comment:
I don't think `TileProvider` needs to be aware of lighting.
Maybe you could move lights handling in NodeProcess?